index.js

```diff
@@ -183,8 +185,38 @@ class Applesign {
   async removePlugins () {
     fchk(arguments, []);
     const plugdir = path.join(this.config.appdir, 'PlugIns');
+    let tests = [];
```

Comment:
can you move the logic of this plugin saver logic into a separate function that returns this array of tests. same goes for the other block below

Reply:
also i think this logic must be done only when the -x is not passed and -p is passed, no need to traverse the files and rename stuff unnecessarily

Reply:
Done!